datasafe-types/src/main/java/de/adorsys/datasafe/business/api/types/utils/LogHelper.java

```diff
@@ -0,0 +1,32 @@
+package de.adorsys.datasafe.business.api.types.utils;
+
+import lombok.SneakyThrows;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.util.Base64;
+
+public class LogHelper {
+    private static String secureLogs = System.getProperty("SECURE_LOGS");
+
+    @SneakyThrows
+    public static String secure(Object value) {
```

Comment:
We need to keep path separators somehow (for URI)

Reply:
done